picard/tagger.py

```diff
@@ -274,6 +274,7 @@ def exit(self):
         self.stopping = True
         self._acoustid.done()
         self.thread_pool.waitForDone()
+        self.save_thread_pool.waitForDone()
```

Comment:
I did a test, inserting a 60s sleep in `file._save_and_rename()` with and without this line.

I load a file, tag it, press Save and exit Picard.
Without this line, the file isn't written to new destination (i put the sleep at start of function), Picard exits totally.

With this line, the file is written to new destination, Picard UI is closed immediatly on quit action, but it stays as a processus (without UI), until the delay expires.

In the first case, i'm not sure when and where the save process is stopped. In the second, we are pretty sure it goes to the end.

I'm worried though about possibly stuck save threads (and therefore stuck Picard processes), so i went the evil way, i ran one Picard, save a file (with sleep), quit, and run another Picard, load/save the same file (possible, sleep is at the start, file isn't moved yet), and when the save finally occurs i get an exception in `info = os.stat(encoded_old_filename)` file not found of course, which is expected.

So :+1: